exchanges/gateio/gateio_websocket.go

```diff
@@ -11,8 +11,10 @@ import (
 	"net/http"
 	"strconv"
 	"strings"
+	"text/template"
 	"time"
 
+	"github.com/Masterminds/sprig/v3"
```

Comment:
I'm just commenting here because this dependency isn't mentioned in this PR or its "stacked dependency" reference.
This is used here to access the function "hasPrefix" when generating subscription templates. 

Only thing I'll say is that we're adding a lot of dependencies when we could just add the following to your funcmap
```go
"hasPrefix":           func(substr string, str string) bool { return strings.HasPrefix(str, substr) },` 
```

Maybe I'll see more usages for it in other PRs. I'll see what you say

Reply:
> Maybe I'll see more usages for it in other PRs. I'll see what you say

Exactly that. In isolation some of the sprig usages seem "_meh_", but taken as a whole they add up to "yeah, that's worth it". 
I think it's used in 4 of them, and it's `mustToJson` was the main driver.

Reply:
PR updated to mention it.

Reply:
@gloriousCode Just spotted this PR is tickless. Like a domestic cat.
Unresolving conversation.
If you're happy with this then approve 😄